plugins/logitech-bulkcontroller/data/lsusb_rallybar.txt

```diff
@@ -0,0 +1,1604 @@
+
+Bus 002 Device 009: ID 046d:089b Logitech, Inc. USB3.2 Hub
+Device Descriptor:
+  bLength                18
+  bDescriptorType         1
+  bcdUSB               3.20
```

Comment:
I don't think you need to include this file - it's really only useful for devices we reverse engineered.

Reply:
Removed